modules/security-context-constraints-about.adoc

```diff
@@ -156,7 +153,22 @@ The restricted SCC is the most restrictive of the SCCs that ship by default with
 * `seccompProfile` is set to `runtime/default` by default.
 * `allowPrivilegeEscalation` must be unset or set to `false` in security contexts.
 
+This is the most restrictive SCC provided by a new installation and will be used by default for authenticated users. [^1^]
```

Comment:
`the most restrictive of the SCCs that ship by default with the system.` would still hold here - it is even the most restrictive SCC for upgraded clusters.

I think just sticking with the note below should be fine.